admin/class-metabox.php

```diff
@@ -890,33 +893,27 @@ public function enqueue() {
 		}
 
 		if ( $pagenow == 'edit.php' ) {
-			wp_enqueue_style( 'edit-page', plugins_url( 'css/edit-page' . WPSEO_CSSJS_SUFFIX . '.css', WPSEO_FILE ), array(), WPSEO_VERSION );
+			$asset_manager -> enqueue_style( 'edit-page' );
 		}
 		else {
 
 			if ( 0 != get_queried_object_id() ) {
 				wp_enqueue_media( array( 'post' => get_queried_object_id() ) ); // Enqueue files needed for upload functionality.
 			}
-			wp_enqueue_style( 'metabox', plugins_url( 'css/metabox' . WPSEO_CSSJS_SUFFIX . '.css', WPSEO_FILE ), array(), WPSEO_VERSION );
-			wp_enqueue_style( 'featured-image', plugins_url( 'css/featured-image' . WPSEO_CSSJS_SUFFIX . '.css', WPSEO_FILE ), array(), WPSEO_VERSION );
-			wp_enqueue_style( 'jquery-qtip.js', plugins_url( 'css/jquery.qtip' . WPSEO_CSSJS_SUFFIX . '.css', WPSEO_FILE ), array(), '2.2.1' );
+			$asset_manager -> enqueue_style( 'metabox-css' );
+			$asset_manager -> enqueue_style( 'featured-image' );
+			$asset_manager -> enqueue_style( 'jquery-qtip.js' );
 
 			// Always enqueue minified as it's not our code.
```

Comment:
This comment should be removed to the place the script is registered (in the asset manager).